sceval_frontend/src/containers/SensorModule.tsx

```diff
@@ -91,7 +91,7 @@ export const SensorModule = withRouter((props: SensorModuleProps & RouteComponen
   const [realtimeMode, setRealtimeMode] = useState<boolean>(false);
 
   // to keep track of component mounted/unmounted event so we don't call set state when component is unmounted
-  let componentUnmounted: boolean;
+  const componentUnmounted = useRef<boolean>(false);
```

Comment:
using `let componentUnmounted: boolean;` cause a warning about this variable being reset on every render so we need to use `useRef` instead.